akka-actor-typed-tests/src/test/scala/akka/actor/typed/SupervisionSpec.scala

```diff
@@ -315,6 +316,18 @@ class SupervisionSpec extends ScalaTestWithActorTestKit(
       }
     }
 
+    "stop when strategy is stop - exception in setup" in {
```

Comment:
isn't this true also without stop strategy, with restart?

Reply:
Yes, only difference is the log. Maybe that shouldn't be logged as it'll the the cause of the ActorIntializationException